docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-node.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Build a Node.js application that uses YSQL
+headerTitle: Build a Node.js application
+description: Build a simple Node.js application using the driver and using the YSQL API to connect to and interact with a Yugabyte Cloud cluster.
+menu:
+  latest:
+    parent: cloud-build-apps
+    name: Node.js
+    identifier: cloud-node
+    weight: 400
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+The following tutorial shows a small [Node.js application](https://github.com/yugabyte/yugabyte-simple-node-app) that connects to a YugabyteDB cluster using the [node-postgres module](../../../../reference/drivers/ysql-client-drivers/#node-postgres) and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in Node.js.
+
+## Prerequisites
+
+In addition to the latest version of [Node.js](https://nodejs.org/en/download/), this tutorial requires the following.
+
+### Yugabyte Cloud
+
+- You have a cluster deployed in Yugabyte Cloud. To get started, use the [Quick start](../../).
+- You downloaded the cluster CA certificate. Refer to [Download your cluster certificate](../../../cloud-secure-clusters/cloud-authentication/#download-your-cluster-certificate).
```

Comment:
You could probably mention that the CA certificate must be downloaded for YSQL.

Reply:
YSQL or YCQL, same certificate